docs/crocks/Identity.md

```diff
@@ -0,0 +1,63 @@
+# Identity
+
+`Identity : a -> a`
+
+It's just a function which returns what you pass into it.
+
+## What a crock!
+
+It converts your `non-crock` into a `crock`
+
+```js
+const frozenPizza = Identity('pizza') //Identity pizza
+const microwave = x => 'microwave ' + x
+const dinner = frozenPizza.map(microwave) //Identity "microwave pizza"
+dinner.value() // "microwave pizza"
+```
+
+| Constructor | Instance |
+|:---|:---|
+| [`of`](#of) | [`ap`](#ap), [`chain`](#chain), [`equals`](#equals), [`map`](#map), [`of`](#of), [`sequence`](#sequence), [`traverse`](#traverse), [`value`](#value) |
+
+## Constructors
+
+### of
+
+```js
+crocks.Identity.of([1,2,10])
+crocks.Identity([1,2,10])
+```
+
+## Instances
+
+### ap
+
+`Function (a -> b) -> Function b`
+
+### chain
+
+`(a -> m b) -> m b`
+
```

Comment:
Here `m` is used to represent any `Monad`:
`Monad m => (a -> m b) -> m b`.

So as this is specifically an Identity, it _could_ (and maybe _should_) be written as:
`(a -> Identity b) -> Identity b` 

Although it would probably look better as: 
`Identity m => (a -> m b) -> m b`

If that makes sense?